__tests__/help.ts

```diff
@@ -1,4 +1,6 @@
-import { exec } from 'child_process';
+import util from 'util';
+
+const exec = util.promisify(require('child_process').exec);
```

Comment:
As per the comments in #15, does this work for `stderr` (the third argument to `exec`)?

Reply:
Yes, it does. Refer to the test case that has been added.